### ISSUE TITLE
corrupted image

### ISSUE THREAD

**Issue by minglich** [lllyasviel/Fooocus#2117]
The process of image generation is normal until the final step, where it turns into a corrupted image.
I can't determine whether it's a graphics card issue or a settings problem.
The final generated images are similar to the two examples below.
![image (1)](https://github.com/lllyasviel/Fooocus/assets/38745988/bf348aa2-2ec5-4886-8192-95070922496c)
![image](https://github.com/lllyasviel/Fooocus/assets/38745988/d69c1d19-1f22-4f58-b1e9-a12b83135014)

**Comment by mashb1t** [comment]
This might be a graphics card issue or an issue with a LoRA. Please make sure if the issue still exists on a fresh install of Fooocus.
Please provide the full terminal output for further analysis.

**Comment by minglich** [comment]
> 这可能是显卡问题或 LoRA 问题。请确保全新安装的 Fooocus 上问题是否仍然存在。 请提供完整的终端输出以供进一步分析。

<img width="1110" alt="2" src="https://github.com/lllyasviel/Fooocus/assets/38745988/b6cd1a75-5c50-458e-95f4-059296743940">

[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 7931184182681290339
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] 1 girl, full color, intricate, elegant, highly detailed, sharp focus, cute, dramatic light, attractive, bright background, professional, thought, iconic, fine detail, clear aesthetic, cool, creative, ambient, expressive, positive, innocent, pure, rational, beautiful, very coherent, pleasing, charismatic, perfect composition, artistic, best,,, winning, pretty
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] 1 girl, confident, full focus, cinematic, complex, glowing, highly detailed, elegant, modern, shiny, rich, color, cool, strong, epic, beautiful, symmetry, illuminated, sharp, incredible, fine detail, intricate, brilliant, awesome composition, colorful, polished, light, iconic, fantastic colors, wonderful, very inspirational, pretty, focused, creative
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.11 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 1.37 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.06 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:04<00:00,  6.68it/s]
Requested to load AutoencoderKL
Loading 1 new model
Image generated with private log at: F:\Fooocus_win64_2-1-831\Fooocus\outputs\2024-02-01\log.html
Generating and saving time: 6.05 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.65 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:04<00:00,  6.45it/s]
Requested to load AutoencoderKL
Loading 1 new model
Image generated with private log at: F:\Fooocus_win64_2-1-831\Fooocus\outputs\2024-02-01\log.html
Generating and saving time: 5.83 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.25 seconds
Total time: 15.17 seconds

**Comment by mashb1t** [comment]
Thos is not the full output, the entrypoint and startup log is missing. Which GPU and which arguments are you using?

**Comment by minglich** [comment]
4090, all settings are default and have been reinstalled.

Full Console Log

`F:\Fooocus_win64_2-1-831>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.1.864
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
Total VRAM 23028 MB, total RAM 65292 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce RTX 4090 : native
VAE dtype: torch.bfloat16
Using pytorch cross attention
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids'}
Base model loaded: F:\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors
Request to load LoRAs [['sd_xl_offset_example-lora_1.0.safetensors', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [F:\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors].
Loaded LoRA [F:\Fooocus_win64_2-1-831\Fooocus\models\loras\sd_xl_offset_example-lora_1.0.safetensors] for UNet [F:\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.40 seconds
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 6125660883020683902
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] 1 girl, intricate, very bright colors, cinematic, dramatic light, elegant, highly detailed, clear background, sharp focus, great composition, perfect dynamic, atmosphere, professional, fine detail, sincere, beautiful, confident, color, winning, epic, gorgeous, creative, positive, attractive, artistic, loving, emotional, pretty, determined, passionate, full, cheerful, vibrant
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] 1 girl, full background, epic, highly detailed, professional fine detail, intricate, elegant, complex, glowing colors, beautiful, sharp focus, incredible light, pretty, attractive, symmetry, striking, iconic, stunning, perfect, creative, best, colorful, contemporary, romantic, futuristic, inspired, vibrant, astounding, coherent, cute, color, clear, brilliant, shiny
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.11 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 1.73 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.28 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:04<00:00,  6.27it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.10 seconds
Image generated with private log at: F:\Fooocus_win64_2-1-831\Fooocus\outputs\2024-02-01\log.html
Generating and saving time: 6.71 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.62 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:04<00:00,  6.67it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.10 seconds
Image generated with private log at: F:\Fooocus_win64_2-1-831\Fooocus\outputs\2024-02-01\log.html
Generating and saving time: 5.65 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.25 seconds
Total time: 16.13 seconds`

**Comment by mashb1t** [comment]
Please check if somehow your VAE is corrupt by renaming it. Fooocus should download then thr VAE again.

**Comment by mashb1t** [comment]
Closing as stale, feel free to reopen/comment if still needed.
